vpos/sdk/python/bancardconnectorpython/api.py

```diff
@@ -0,0 +1,493 @@
+# MIT License
+#
+# Copyright (c) [2018] [Victor Manuel Cajes Gonzalez - vcajes@gmail.com]
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+
+import os
+import json
+import hashlib
+import requests
+from bancardconnectorpython.util import *
+from bancardconnectorpython.exceptions import *
+
+
+class BancardAPI(object):
+	# possibles values for the environment class member
+	ENVIRONMENT_SANDBOX = "sandbox"
+	ENVIRONMENT_PRODUCTION = "production"
+
+	# currently PYG is the only allowed currency for Bancard
+	BANCARD_ALLOWED_CURRENCIES = ["PYG"]
+
+	# URL keys
+	ROLLBACK_KEY = "rollback"
+	CHARGE_TOKEN_GENERATOR_KEY = "single_buy"
+	PAYMENT_WEB_URL_KEY = "payment"
+	CONFIRMATIONS_KEY = "confirmation"
+
+	BANCARD_BASE_URL_SANDBOX = "https://vpos.infonet.com.py:8888"
+	BANCARD_BASE_URL_PRODUCTION = "https://vpos.infonet.com.py"
+
+	SANDBOX_URLS = {
```

Comment:
what do you think about moving all this constants into a constants' file to be used here by importing it instead of having all of them defined here?

Reply:
I agree with you, it will improve the modularity of the connector. I'm going to make the change.